app-frontend/config/webpack.dev.config.js

```diff
@@ -0,0 +1,29 @@
+var webpack = require('webpack'),
+    config = require('./webpack.common.config.js'),
+    host = process.env.WEBPACK_DEV_SERVER || 'http://localhost:8080';
+
+config.devtool = 'inline-source-map';
```

Comment:
eval messes with the source maps (outputs javascript instead of es6), and inline-source-map seems to work the best as far as breakpoints.